src/Microsoft.Restier.Security/RoleBasedAuthorization.cs

```diff
@@ -17,28 +17,10 @@
 namespace Microsoft.Restier.Security
 {
     /// <summary>
-    /// Represents a role-based authorization system.
+    /// This class applies security policy on expanded expression nodes.
     /// </summary>
-    public class RoleBasedAuthorization : IQueryExpressionInspector,
-        IQueryExpressionExpander, IDelegateHookHandler<IQueryExpressionExpander>
+    public class ApiPolicyActivator : IQueryExpressionExpander
```

Comment:
I think it's accurate to describe the class here. Please go ahead. By the way, to keep consistency, I think `AssertAttribute` should be renamed to `AssertRoleAttribute` and `RoleBasedAuthorization` should be renamed to `RoleBasedAuthorizer`. But that's OK if you feel this should be a separate task.